tests/integration/test_completion.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import textwrap
+from pathlib import Path
+
+import craft_cli
+from craft_cli.completion import complete
+
+
+class TestLsCommand(craft_cli.BaseCommand):
```

Comment:
```suggestion
class FakeLsCommand(craft_cli.BaseCommand):
```

Likewise for the cp command - pytest uses the `Test` prefix for classes to presume that the class contains tests, and then it complains.